hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -736,6 +745,26 @@ final Stream<DynamicTest> tokenClaimAirdropDisabledFail() {
                         .hasPrecheck(NOT_SUPPORTED));
     }
 
+    @HapiTest
+    @DisplayName("a hollow account receiver that sigs it with no HBAR fail")
+    final Stream<DynamicTest> hollowNoHbarFail() {
+        final var hollowAcnt = "hollowAcnt";
+        return hapiTest(
+                cryptoCreate(OWNER).balance(ONE_HUNDRED_HBARS),
+                createFT(FUNGIBLE_TOKEN_1, OWNER, 1000L),
+                newKeyNamed(hollowAcnt).shape(SECP_256K1_SHAPE),
+                createHollowAccountByFunToken(hollowAcnt, FUNGIBLE_TOKEN_1, OWNER),
+                tokenAirdrop(moving(10, FUNGIBLE_TOKEN_1).between(OWNER, hollowAcnt))
+                        .payingWith(OWNER),
+                getAliasedAccountInfo(hollowAcnt)
+                        .has(accountWith().hasEmptyKey().noAlias()),
```

Comment:
you can replace this with getAliasedAccountInfo(hollowAcnt).isHollow()

Reply:
Thanks @ibankov. I just checked .isHollow(), only checked `assertEquals(IMMUTABILITY_SENTINEL_KEY, key);`. Here I also check .noAlias().